package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@asyncapi/cli",
   "description": "All in one CLI for all AsyncAPI tools",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Since CLI `v2.0.2` has been already released, is this PR still valid? @Souvikns @Shurtu-gal @Amzani

Reply:
I don't think so, we can close this.